packages/kbn-field-types/BUILD.bazel

```diff
@@ -0,0 +1,89 @@
+
+load("@npm//@bazel/typescript:index.bzl", "ts_config", "ts_project")
```

Comment:
Think it could be a good idea to have someone from @elastic/kibana-operations review this. What do you say?